codalab/worker/worker_run_state.py

```diff
@@ -446,6 +502,9 @@ def _transition_from_CLEANING_UP(self, run_state):
                     time.sleep(1)
 
         for dep in run_state.bundle.dependencies:
+            if dep.child_path == '.':
+                continue
+
             dep_key = DependencyKey(dep.parent_uuid, dep.parent_path)
             if not self.shared_file_system:  # No dependencies if shared fs worker
                 self.dependency_manager.release(run_state.bundle.uuid, dep_key)
```

Comment:
Why is this exempt for `.` dependencies?  Seems like it's still a dependency and the only thing that's affected is the mounting behavior.

Reply:
After some more investigation, I found out that I wasn't cleaning up all the content that was mounted at the top. I fixed this in my recent commit.